packer/plugin-getter/github/getter.go

```diff
@@ -156,10 +158,40 @@ func (t *HostSpecificTokenAuthTransport) base() http.RoundTripper {
 	return http.DefaultTransport
 }
 
+type GithubPlugin struct {
```

Comment:
nit: Lets call this what it is a plugin URI for GitHub. 
```suggestion
type GitHubPluginURI struct {
```

Reply:
Hmmmm, since the general structure is called `Plugin`, should we rename both to be `URI`? This would make more sense probably.
I'm hesitant in changing this one without changing the other, if only for consistency's sake.